injecteddll/bindings_dll/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
```

Comment:
Take the bindings from the ones we already have in the root

Reply:
Fixed